helm/templates/ingress.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+    nginx.ingress.kubernetes.io/use-regex: "true"
```

Comment:
Helm charts should either have a properly configurable ingress controller (where ingress class and annotations are passed as templated values), or not have one at all.